rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/dto/DependencyGraphItem.java

```diff
@@ -0,0 +1,86 @@
+package org.jboss.windup.rules.apps.java.reporting.freemarker.dto;
+
+import org.apache.commons.io.FilenameUtils;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.rules.apps.java.archives.model.IdentifiedArchiveModel;
+import org.jboss.windup.rules.apps.java.dependencyreport.DependencyReportDependencyGroupModel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A data transfer object carying information about the dependency
+ * @author <a href="mailto:marcorizzi82@gmail.com>Marco Rizzi</a>
+ */
+public class DependencyGraphItem
+{
+   private final Kind kind;
+   private final Map<String, String> metadata;
+
+   public DependencyGraphItem(DependencyReportDependencyGroupModel dependencyReportDependencyGroupModel)
+   {
+      this(dependencyReportDependencyGroupModel.getCanonicalProject());
+   }
+
+   public DependencyGraphItem(ProjectModel projectModel)
+   {
+      this.metadata = new HashMap<>(1);
```

Comment:
why do we need to use "this." ?

Reply:
A former developer on the project used to complain about member variable references that didn't use "this.". I have a feeling that it became a habit since then for some of us.